src/pred_set/src/pred_setScript.sml

```diff
@@ -965,6 +965,13 @@ val DIFF_UNION = store_thm(
 ``!x y z. x DIFF (y UNION z) = x DIFF y DIFF z``,
 SRW_TAC[][EXTENSION] THEN METIS_TAC[])
 
+Theorem UNION_DIFF_EQ:
```

Comment:
Please make this an automatic rewrite (add `[simp]` after theorem name), and add an extra conjunct with the  union the other way round.